r/R/rzeppelin/R/zzz.R

```diff
@@ -0,0 +1,9 @@
+typeMap <- list()
```

Comment:
Sure.  rscala is a two-way connection, but we only need to use one-way functionality.  So I didn't include the rscala files that create the connection in the opposite direction.  This file was modified to remove a reference to a function in one of those files, which is no longer used.  

The change -- the deletion of two lines -- is too small to assert any copyright interest in the modification.  It remains entirely under BSD.